api/namex/models/word_classification.py

```diff
@@ -0,0 +1,49 @@
+""""word classification classifies all words in a name approved by an exmainer to be used for auto-approval
+
+"""
+from . import db, ma
+from datetime import datetime
+from sqlalchemy.orm import backref
+
+
+class WordClassification(db.Model):
+    __tablename__ = 'word_classification'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    classification = db.Column('word_classification',db.String(4),default='NONE',nullable=False,index=True)
+    word = db.Column('word', db.String(1024), nullable=False, index=True)
+    lastNameUsed = db.Column('last_name_used',db.String(1024))
+    lastPrepName = db.Column('last_prep_name',db.String(1024))
+    frequency = db.Column('frequency', db.BIGINT)
+    approvedBy = db.Column('approved_by', db.Integer, db.ForeignKey('users.id'))
+    approvedDate = db.Column('approved_dt', db.DateTime(timezone=True), default=datetime.utcnow)
+    startDate = db.Column('start_dt', db.DateTime(timezone=True), default=datetime.utcnow)
+    endDate = db.Column('end_dt', db.DateTime(timezone=True), default=datetime.utcnow)

```

Comment:
When you first create a record, this will populate the endDate value with the current date. Do you want that? If you leave this in, and you want endDate to be blank upon insert, you will have to specifically say endDate=None. Same case with approvedDate above but I'm not sure I know the functionality around that one.